packages/auth/src/authClients/firebase.ts

```diff
@@ -24,12 +32,34 @@ export const firebase = (client: Firebase): AuthClient => {
     type: 'firebase',
     client,
     restoreAuthState: () => client.auth().getRedirectResult(),
-    login: async (usingProvider: oAuthProvider = 'google.com') => {
+    login: async (
+      usingProvider: oAuthProvider | PasswordProvider = 'google.com'
```

Comment:
I think for readability, what I would do in this case is change the login param name to be just `with`

So like 

```
login: async(with: oAuthProvider | PasswordCreds | EmailLink)
```
I think `PasswordProvider` is slightly misleading

Just my 2p though 👍

Reply:
I did `withAuth`. (`with` is a keyword in typescript.)